docs/src/operate/c8y/cloud-profiles.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Cloud Profiles
+tags: [Operate, Cumulocity, Cloud Profile]
+description: Connecting %%te%% to multiple Cumulocity tenants
+---
+
+import UserContext from '@site/src/components/UserContext'; import
+UserContextForm from '@site/src/components/UserContextForm';
+
+**%%te%%** now supports multiple Cumulocity connections using cloud profiles.
+This can be useful for migrating from one Cumulocity tenant to another, where
```

Comment:
While this is also a genuine use-case, I thought the primary use case is to keep the device connected to multiple tenants parallelly for different kinds of data. E.g: Connected to an edge instance for faster edge telemetry processing and a cloud instance for remote device management. Do we want to promote the feature like that or focus just on migration cases for now(until it is battle tested)?

Reply:
yeah we could, but we can always add additional use cases post release